crates/store/re_grpc_client/src/lib.rs

```diff
@@ -376,3 +387,104 @@ async fn stream_catalog_async(
 
     Ok(())
 }
+
+// Craft a blueprint from relevant chunks and activate it
+fn activate_catalog_blueprint(
+    tx: &re_smart_channel::Sender<LogMsg>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    let blueprint_store_id =
+        StoreId::from_string(StoreKind::Blueprint, "catalog_blueprint".to_owned());
+    let blueprint_store_info = StoreInfo {
+        application_id: ApplicationId::from("redap_catalog"),
```

Comment:
This prob needs to be the same as the recording store, best make sure it's based on some constant or something.